prprocessor/__main__.py

```diff
@@ -26,6 +26,7 @@
     re.IGNORECASE,
 )
 COMMIT_ISSUES_REGEX = re.compile(r'#(\d+)')
+EXEMPT_COMMIT_REGEX = re.compile(r'^i18n\s*-', re.IGNORECASE)
```

Comment:
I think we can be more specific here to keep them standardized. Since they're typically generated by scripts that should be fine.
```suggestion
EXEMPT_COMMIT_REGEX = re.compile(r'^i18n - (extracting new, )?pulling from tx$', re.IGNORECASE)
```

Reply:
Changed.